tests/test_utils_reqser.py

```diff
@@ -87,6 +93,9 @@ def parse_item(self, response):
     def handle_error(self, failure):
         pass
 
+    def __parse_item_private(self, response):
```

Comment:
Could you maybe add a test that accounts for methods that do not match the requirements? More than one trailing underscore, for instance.

Reply:
Ah, commented above but adding a test for the regex directly to make sure it matches/doesn't match variously underscored names.